### ISSUE TITLE
Enabling CAP causes non-guest-authors who have authored posts to show different post counts (WordPress 4.9.2)

### ISSUE THREAD

**Issue by lidocaine** [Automattic/Co-Authors-Plus#494]
Originally mentioned in Issue #470 by @benlk 

> ...in WordPress 4.9.2. Enabling CAP causes non-guest-authors who have authored posts to show different post counts...

and

> With CAP enabled, `nkhan` has 248 posts listed at `/wp-admin/users.php`. With CAP disabled, `nkhan` has 385 posts - is this because the posts created by `nkhan` but filed under a different byline are being excluded from `nkhan`'s post count in the users table? If it is, that's not a problem.

Possibly related to Issues #447 and #231

**Comment by jeherve** [comment]
I was able to reproduce this, but I believe this is done on purpose: CAP unsets the default post count in that column and replaces it with its own count since this changeset:
https://plugins.trac.wordpress.org/changeset/479608/

CAP' custom post count happens here:
https://github.com/Automattic/Co-Authors-Plus/blob/3.2.2/co-authors-plus.php#L937-L952

If a user has a linked guest author, the post count is updated to return the number of posts by that guest author instead of showing the number of posts by the default user.

I can see how this could be confusing though. Maybe we should be more clear about the contents of that column when CAP is active? One way we could make it clearer would be to add a new column to the Users table, that would list guest authors when there are some. 

I've opened #502 to make that suggestion. I'd be happy to hear feedback to see if this makes things less confusing, or if there are other things we could do to make thing less confusing.
